swift/ql/lib/swift/File.qll

```diff
@@ -0,0 +1,24 @@
+/** TODO: to be replaced */
+
+/** A file. */
+class File extends @file {
+  /** toString */
```

Comment:
Nit (mostly just curious): Are the useless comments to silence a linter?

Reply:
Yes, there is a CI check for this, which we may turn off for the time being.